src/_includes/side_nav.html

```diff
@@ -9,7 +9,7 @@
           <li class="related__section-link">
               {% assign grouped_related_pages = site.pages | children_of: item | group_by: "related_order" | sort: "name" %}
               {% if grouped_related_pages.size > 0 %}
-                <a class="{% if item.url == page.url %}current{% endif %}" href="{{ item.url | relative_url }}">{{ item.title }}</a>
+                <a class="{% if item.url == page.url %}current{% endif %}" href="{{ item.url | relative_url }}" {% if item.url == page.url %}aria-current="page"{% endif %}>{{ item.title }}</a>
```

Comment:
Could probably simplify these so that only one conditional is needed per line?

```html
<a {% if item.url == page.url %}class="current" aria-current="page"{% endif %} href="{{ item.url | relative_url }}">{{ item.title }}</a>

```